### DISCUSSION TITLE
Is there a way to know where my free Copilot come from?

### DISCUSSION THREAD

**Discussion by TigerBeanst** [community/community#35506] [Copilot Conversations] [answered]
I currently get Copilot for free, but I would like to know why I got it, from which repo

**Comment by davecheney** [comment, marked as answer]
@TigerBeanst you had a student coupon on your account when you activated Copilot.

**Reply by TigerBeanst** [reply]
But my student coupon have expired...

**Reply by davecheney** [reply]
It was not expired when you applied.

**Comment by clraxiinaxmed837-cloud** [comment]
Tttt
